openmetadata-service/src/main/java/org/openmetadata/service/resources/databases/TableResource.java

```diff
@@ -905,6 +906,53 @@ public Response deleteFollower(
         .toResponse();
   }
 
+  @PUT
+  @Path("/{fqn}/lifeCycle")
+  @Operation(
+      operationId = "addLifeCycleData",
+      summary = "Add life cycle data",
+      description = "Add life cycle data to the table.",
+      responses = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "The table with the new life cycle updates",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = Table.class))),
+        @ApiResponse(responseCode = "400", description = "Bad request")
+      })
+  public Table addLifeCycle(
+      @Context UriInfo uriInfo,
+      @Context SecurityContext securityContext,
+      @Parameter(description = "Fully qualified name of the table", schema = @Schema(type = "string")) @PathParam("fqn")
+          String fqn,
+      @Valid LifeCycle lifeCycle) {
+    OperationContext operationContext = new OperationContext(entityType, MetadataOperation.EDIT_LIFE_CYCLE);
+    authorizer.authorize(securityContext, operationContext, getResourceContextByName(fqn));
+    Table table = repository.addLifeCycle(fqn, lifeCycle);
+    return addHref(uriInfo, table);
+  }
+
+  @GET
+  @Path("/{fqn}/lifeCycle")
```

Comment:
instead of having another path we can just retrieve this via `fields=lifecycle` when doing a normal GET